day2/test/test_gift.py

```diff
@@ -0,0 +1,27 @@
+from unittest import TestCase, main
+from day2.gift import Gift
+
+
+class Test_gift(TestCase):
```

Comment:
Acho que poderia adicionar mais testes aí!
E se a lista de `dimensions` vier vazia? Se tiver apenas duas dimensões? Se tiver todos os número iguais? Se for número negativo?